packages/cascara/src/ui/Table/TechnicalDesign.mdx

```diff
@@ -142,3 +142,34 @@ For mutation purposes, the implementation will support `select`, `input`, `boole
+
```

Comment:
haha holy crap. That is a lot. This is going to make it difficult to work with our files if we have to jump over big data IDs. I wonder if there is a way to define these big chunks of data as reusable links at the foot of a MDX file. That way if we DO want to do this and not just place a file relative to the file itself, the big chunks of base64 are at the bottom of the document. 

- [ ] Not sure if reusable links will work with base64 images but could we try it?

Reply:
nahh, it didn't work 😞 
but, we now have svg diagrams!